src/icons/src/IconWrapper.js

```diff
@@ -1,13 +1,14 @@
 import React, { forwardRef, memo } from 'react'
 import PropTypes from 'prop-types'
 import ReactIs from 'react-is'
+import Box from 'ui-box'
 
 /**
  * This is an internal helper component for rendering custom or Evergreen icons
  * Box props are applied to the outer Box container, and Evergreen icon-specific props are added to the icon element.
  */
 export const IconWrapper = memo(
-  forwardRef(function Icon({ icon, ...props }, ref) {
+  forwardRef(function Icon({ icon, marginLeft, marginRight, marginTop, marginBottom, ...props }, ref) {
```

Comment:
It might be better to pull out all `icon`-specific props here and pass those down instead of `...props` (e.g. for `React.cloneElement`, etc).

Then we can pass `...props` directly to the outer Box.